src/modules/order/components/OrderCheckout/OrderCheckout.tsx

```diff
@@ -23,7 +19,7 @@ export const OrderCheckout: React.FC<OrderCheckoutProps> = ({
             min={1}
             css={styles.quantity}
             onChange={(e) => {
-              onQuantity(parseInt(e.target.value));
+              setQuantity(parseInt(e.target.value));
```

Comment:
You should use currrentTarget instead of target:
https://thisthat.dev/current-target-vs-target/
https://www.jstips.co/en/javascript/difference-between-target-and-currentTarget/